apps/prairielearn/src/ee/lib/ai-grading.ts

```diff
@@ -375,6 +375,7 @@ export async function aiGrade({
               // NEXT STEPS: rubrics
             },
             user_id,
+            true,
```

Comment:
Like line 371, it'd be nice to have a trailing comment stating what this is:

```suggestion
            true, // is_ai_graded
```

This function is up to 7 arguments, so it may be a good candidate to switch from individual arguments to an object with named properties. I won't request that change here. cc @jonatanschroeder, this might be a nice refactor sooner or later.